### ISSUE TITLE
[Verified] Ban: Older Items link disappears if user is blocked by someone previously present in their homefeed

### ISSUE THREAD

**Issue by smallq-git** [pepyatka/pepyatka-html#578]
Pre-conditions: 
1. user1 is subscribed to user2.
2. user2 was subscribed to user1, liked/commented on their posts, posted to groups visible to user1.
1. As user2 block user1.
2. As user1 log in and scroll to the bottom of homefeed.

Expected: Older Items link is present like before.
Actual: Only one page is available.

**Comment by smallq-git** [comment]
This issue exists on live.

**Comment by smallq-git** [comment]
This is fixed - I'm guessing by a8875ec

**Comment by epicmonkey** [comment]
@smallq-git I'd keep this open until we get a proper fix. Right now Older entries link is visible even on the last page. It's up to you tho' if you want to create a new issue.

**Comment by smallq-git** [comment]
https://github.com/pepyatka/pepyatka-html/issues/611 should serve as a reminder.
